openstack/sre/templates/prometheus-alerts.yaml

```diff
@@ -13,48 +13,24 @@ spec:
   groups:
   - name: openstack-sre.alerts
     rules:
-    - alert: OpenstackSRE2or5PctErrorBudgetBurn
-      expr: |2
-          (
-            (100 * global:api_errors_per_request_sli:ratio_rate1h > on (ingress) 14.4 * global:api_errors_per_request_slo:percent)
-            and
-            (100 * global:api_errors_per_request_sli:ratio_rate5m > on (ingress) 14.4 * global:api_errors_per_request_slo:percent)
-          )
-          or
-          (
-            (100 * global:api_errors_per_request_sli:ratio_rate6h > on (ingress) 6 * global:api_errors_per_request_slo:percent)
-            and
-            (100 * global:api_errors_per_request_sli:ratio_rate30m > on (ingress) 6 * global:api_errors_per_request_slo:percent)
-          )
-      for: 2m
+    - alert: ApiAvailabilityWarning
+      expr: (1 - global:api_errors_per_request_sli:ratio_rate1h) *100 <= on (ingress,region) global:api_availability_warning_slo:percent
       labels:
         context: availability
         service: "{{`{{$labels.ingress}}`}}"
-        severity: info
+        severity: warning
         tier: sre
       annotations:
-        summary: "an api burns its availability error budget very fast"
-        description: "{{`{{$labels.ingress}}`}} api has returned {{`{{$value}}`}}% 5xx."
+        summary: "An API fell below availability warning SLO in the last hour"
+        description: "{{`{{$labels.ingress}}`}} fell below availability warning SLO in the last hour."
 
-    - alert: OpenstackSRE10PctErrorBudgetBurn
-      expr: |2
-          (
-            (100 * global:api_errors_per_request_sli:ratio_rate1d > on (ingress) 3 * global:api_errors_per_request_slo:percent)
-            and
-            (100 * global:api_errors_per_request_sli:ratio_rate2h > on (ingress) 3 * global:api_errors_per_request_slo:percent)
-          )
-          or
-          (
-            (100 * global:api_errors_per_request_sli:ratio_rate3d > on (ingress) global:api_errors_per_request_slo:percent)
-            and
-            (100 * global:api_errors_per_request_sli:ratio_rate6h > on (ingress) global:api_errors_per_request_slo:percent)
-          )
-      for: 1h
+    - alert: ApiAvailabilityCritical
+      expr: global:api_errors_per_request_sli:ratio_rate1h <= on (ingress,region) global:api_availability_critical_slo:percent
       labels:
         context: availability
         service: "{{`{{$labels.ingress}}`}}"
-        severity: info
+        severity: warning
```

Comment:
Why just warning here? It's supposed to be critical in the end, right?

Reply:
true